src/main/java/org/terracotta/build/plugins/docker/DockerBuildPlugin.java

```diff
@@ -45,7 +47,9 @@ public void apply(Project project) {
 
     DockerBuildExtension buildExtension = ((ExtensionAware) dockerExtension).getExtensions().create("build", DockerBuildExtension.class);
 
-    configureBuildDefaults(project, buildExtension);
+    BuildInfoExtension buildInfoExtension = project.getExtensions().getByType(BuildInfoExtension.class);
```

Comment:
If we're going to unconditionally use the extension here we should probably have the `DockerBuildPlugin` apply the `BuildInfoPlugin`.